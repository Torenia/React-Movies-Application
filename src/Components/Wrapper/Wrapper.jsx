import styled from 'styled-components';

export const Wrapper = styled.div`
  padding: 0 10%;
  background-color: #232323;
`;

export const ErrorBoundaryWrapper = styled(Wrapper)`
  height: calc(100vh - 350px);
  display: flex;
  justify-content: center;
  align-items: center;
  
  h2 {
    margin: 0;
    color: #f00;
  }
`;

