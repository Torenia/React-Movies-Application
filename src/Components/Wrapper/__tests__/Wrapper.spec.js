import React from 'react';

import { Wrapper, ErrorBoundaryWrapper } from '../Wrapper';

describe('<Wrapper />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Wrapper />);
  });

  it('should render wrapper', () => {
    expect(wrapper).toMatchSnapshot();
  });
});

describe('<ErrorBoundaryWrapper />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<ErrorBoundaryWrapper />);
  });

  it('should render error boundar wrapper', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
