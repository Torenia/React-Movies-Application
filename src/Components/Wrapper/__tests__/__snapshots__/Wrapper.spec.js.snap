// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorBoundaryWrapper /> should render error boundar wrapper 1`] = `
.c0 {
  padding: 0 10%;
  background-color: #232323;
  height: calc(100vh - 350px);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0 h2 {
  margin: 0;
  color: #f00;
}

<div
  className="sc-AxjAm c0"
/>
`;

exports[`<Wrapper /> should render wrapper 1`] = `
.c0 {
  padding: 0 10%;
  background-color: #232323;
}

<div
  className="c0"
/>
`;
